packages/paste-style-props/src/types/flexbox.ts

```diff
@@ -29,15 +29,7 @@ export type AlignContent = ResponsiveValue<AlignContentOptions>;
  *
  * [MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items)
  */
-export type JustifyItemsOptions =
-  | CSS.Globals
-  | CSS.SelfPosition
-  | 'baseline'
-  | 'left'
-  | 'legacy'
-  | 'normal'
-  | 'right'
-  | 'stretch';
+export type JustifyItemsOptions = Property.JustifyItems;
```

Comment:
Not intentional no, but it does raise the question of what is the desired outcome, because there are a number of other CSS types that also currently allow `string` or `null` in what we've already shipped. `fontStyle` for example.

I can try to replicate what we have with these flex types, but I fear we may have already done a whoopsie elsewhere.

Reply:
Annoyingly, they moved all the things like ContentDistribution, ContentPosition, SelfPosition, to a private namespace they don't export. So we need to do a bit of extra work to replicate this